### ISSUE TITLE
Ensure user understands search results may not be complete and more tools exists

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/arctos#6019]
Currently in the results:

![Firefox_Screenshot_2023-03-16T22-07-32 349Z](https://user-images.githubusercontent.com/2523089/225773278-886ae282-b144-4484-9da0-6baf6aa2ab2b.png)

Two changes:
1) Change the button ```Customize Results``` to ```Result Options```

2) Instead of above, move the ```record limit:250000``` to the line below so the new phrasing would be:

```Showing 1 to 100 of 250,000 entries``` and then a box (green? red seems too harsh) or in bold: ```Record limit: 250000 ; more results may exist. Please use Result Options or Tools for more access.```

**Comment by Jegelewicz** [comment]
> Change the button Customize Results to Result Options

We are using "Customize" everywhere else all over Arctos - why do something different here?

**Comment by dustymc** [comment]
Maybe worth discussing my [latest crazy idea](https://github.com/ArctosDB/arctos/issues/6020#issuecomment-1477100440) first, but...

>Change the button 

I'm with Teresa, that seems arbitrary?

>to the line below s

Above is super-easy and can't break any of the built-in functionality. "Below" is part of datatables - I can probably change it if I really have to, but that would be a huge sustainability hit (would move us from running well-established code that thousands of apps use to running one-off code that's all us) along with being less-fun to build.

>Please use Result Options

Is that clear? I imagine that anyone who'll read that already has, and anyone who didn't the first time maybe won't the second either??

> or Tools

I don't know how that would help, please elaborate?

**Comment by dustymc** [comment]
I think this has all been addressed elsewhere, tentatively closing.

**Comment by bryansmclean** [comment]
I have to weigh in again to these threads and say that Im still *flabbergasted* about how to view large sets of search results. 

Right now, I simply want to view all MSB:Mamm collected from Mongolia in 2022. By default (w/ my requested fields) I can only view 250 (less than a fifth of what exists). Do I really have to create a search profile (or choose an existing one) to obtain all the records?

And is this the solution we have arrived at to ensure people around the world who are interested in biodiversity data can use Arctos efficiently? Please fill me in if I missed the latest developments in how to easily view all records from a search.

**Comment by dustymc** [comment]
1) https://github.com/ArctosDB/arctos/issues/6111 potentially provides a method to just get these data (if they can be cached), it's stuck under a bunch of dependencies and a general lack of feedback, I can't do it by myself. HELP!

2)
<img width="395" alt="Screenshot 2023-06-02 at 6 41 22 AM" src="https://github.com/ArctosDB/arctos/assets/5720791/c2df9d8d-f79c-40a0-863c-a8ed2ae25aba">

**Comment by bryansmclean** [comment]
This option seems to be available as an Arctos manager, not a general user?

**Comment by dustymc** [comment]
>This option seems to be available as an Arctos manager, not a general user?

https://github.com/ArctosDB/arctos/issues/6366

But, it seems that 6111 will probably solve this, without any added complexity or increased security risk or etc., for most users in most situations. The solution seem completely straightforward, but there are some (easy?) cleanups in linked issues that would avoid unexpected changes or further interruptions. What looks to me like a pretty good solution has been proposed for what's obviously a major obstacle, I'm having a hard time understanding why I can't get the little bit of help I need to make it reality. So help?!

(And I still have no idea why this issue is open....)

**Comment by mkoo** [comment]
Stale issue alert: The original issue topic has been discussed and some things have been implemented to address, so I am going to close this topic. Please file a new issue to start a fresh thread to reflect the current state of Arctos- thanks!
